frontend-react/src/pages/submissions/SubmissionDetails.tsx

```diff
@@ -104,6 +104,20 @@ function SubmissionDetailsContent() {
     );
     const submissionDate = generateSubmissionDate(actionDetails.submittedAt);
 
+    /* Conditional title strings */
+    const preTitle = `${
+        actionDetails.submitter
+    } ${actionDetails.topic.toUpperCase()} Submissions`;
```

Comment:
Maybe more `?` are needed around here? (e.g. `actionDetails?.topic?.toUpperCase()`) Is there any chance `topic` will be null?

Reply:
Topic _currently_ has no way of being null because all of our topics are `covid-19`. That, and the default is `""` so worst case, it will be whitespace :)

Reply:
"Never trust data coming over the wire", but if the underlying network class always makes sure this cannot be undefined/null then 👍 .